src/components/Toggle/Toggle.scss

```diff
@@ -45,15 +47,15 @@
     &--selected {
       // @include theme-prop(background-color, primary-color);
       background: linear-gradient(180deg, #cefb53 0%, #7af94b 100%);
-      box-shadow: 0px 3px 8px rgba(0, 0, 0, 0.3), 0px 3px 1px rgba(0, 0, 0, 0.05);
+      box-shadow: 0px 3px 8px $black-3, 0px 3px 1px rgba(0, 0, 0, 0.05);
       &::after {
-        left: 32px;
+        left: $spacing-xl;
       }
     }
 
     &--not-selected {
       // @include theme-prop(background-color, ui-border-color);
-      background: rgba(0, 0, 0, 0.2);
+      background: $black-2;
```

Comment:
please in the mixin create for example variable  toggle-primary-bg-color: $black-2 and then use with them-prop